system/Session/Handlers/BaseHandler.php

```diff
@@ -129,9 +129,11 @@
 	 * Constructor
 	 *
 	 * @param BaseConfig $config
+	 * @param string $ipAddress
 	 */
-	public function __construct($config, string $ipAddress)
+	public function __construct(BaseConfig $config, string $ipAddress)
```

Comment:
The vagueness is on purpose. an App() is actually passed in, with the properties that are not in BaseConfig.
It just needs to be a class with the "proper" properties.
It breaks your build big-time, as you notive from travis-ci.